SoObjects/SOGo/SOGoUserManager.m

```diff
@@ -649,7 +659,7 @@ - (BOOL) checkLogin: (NSString *) _login
             [currentSource setBindPassword: _pwd];
           }
     }
-	    
```

Comment:
As mentioned previously, this is not needed. If you think it is, let me know how this can be needed. Otherwise, remove that if block.